proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -130,9 +130,8 @@ message AddActivityTaskRequest {
     string forwarded_source = 7;
     temporal.server.api.enums.v1.TaskSource source = 8;
     temporal.server.api.clock.v1.VectorClock clock = 9;
-    // If using build-id based versioning: version stamp of last worker to complete a workflow
-    // task for this workflow. Used to dispatch to correct worker in matching.
-    temporal.api.common.v1.WorkerVersionStamp worker_version_stamp = 10;
+    // How this task should be directed by matching.
+    temporal.server.api.taskqueue.v1.TaskVersionDirective version_directive = 10;
```

Comment:
Wow we introduces so many concepts with this feature, we have directives, stamps, capabilities, reachability...
Not saying it's a bad thing but we'll need a glossary to explain all of those.

Reply:
I'd add a comment saying what the absence of this directive means.